drivers/macvlan/macvlan.go

```diff
@@ -15,13 +17,17 @@ const (
 	vethLen             = 7
 	containerVethPrefix = "eth"
 	vethPrefix          = "veth"
-	macvlanType         = "macvlan"  // driver type name
-	modePrivate         = "private"  // macvlan mode private
-	modeVepa            = "vepa"     // macvlan mode vepa
-	modeBridge          = "bridge"   // macvlan mode bridge
-	modePassthru        = "passthru" // macvlan mode passthrough
-	parentOpt           = "parent"   // parent interface -o parent
-	modeOpt             = "_mode"    // macvlan mode ux opt suffix
+	macvlanType         = "macvlan"                           // driver type name
+	modePrivate         = "private"                           // macvlan mode private
+	modeVepa            = "vepa"                              // macvlan mode vepa
+	modeBridge          = "bridge"                            // macvlan mode bridge
+	modePassthru        = "passthru"                          // macvlan mode passthrough
+	parentOpt           = "parent"                            // parent interface -o parent
+	modeOpt             = "_mode"                             // macvlan mode ux opt suffix
+	parentFile          = "parent-file"                       // parent file
+	globalScope         = "global"                            // local scope label data
+	localScope          = "local"                             // local scope label data
+	driverPrefix        = "com.docker.network.driver.macvlan" // driver prefix used for labels
```

Comment:
Please add another labelIdentifier as `driverScopeLabel = driverPrefix+".scope"` and use that to check for scope functionality. It will keep way for other labels to come in the future.